src/components/Link.js

```diff
@@ -0,0 +1,4 @@
+import { Link } from 'redux-little-router';
```

Comment:
I can roll these exports back in favor of exporting the whole lib at once in `src/router.js`, but the underlying issue is that redux-little-router creates a 'cache' to store matched routes when it is first imported, and the `<Fragment>` and `<Link>` components in the consumer therefore need to use the _same_ import as the platform. If the consumer imports its own `Link` and `RouterProvider` directly from `redux-little-router` itself, it will get a new cache independent of the platform cache, and things won't work.

It's not a system I particularly like, and it makes me more skeptical of redux-little-router